<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WndMain</class>
 <widget class="QWidget" name="WndMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>576</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>788</width>
    <height>576</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>788</width>
    <height>576</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GameWidget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>637</width>
     <height>77</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>637</width>
     <height>77</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>637</width>
     <height>77</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>152</width>
     <height>576</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>152</width>
     <height>576</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>152</width>
     <height>576</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>152</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>152</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>152</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/image/logo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>116</width>
     <height>56</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>116</width>
     <height>56</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>116</width>
     <height>56</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string> 游 戏</string>
   </property>
   <property name="icon">
    <iconset resource="imandqss.qrc">
     <normaloff>:/image/image/gameIcon.png</normaloff>:/image/image/gameIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>116</width>
     <height>56</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>116</width>
     <height>56</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>116</width>
     <height>56</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string> 记 录</string>
   </property>
   <property name="icon">
    <iconset resource="imandqss.qrc">
     <normaloff>:/image/image/histIcon.png</normaloff>:/image/image/histIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>116</width>
     <height>56</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>116</width>
     <height>56</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>116</width>
     <height>56</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string> 我 的</string>
   </property>
   <property name="icon">
    <iconset resource="imandqss.qrc">
     <normaloff>:/image/image/userIcon.png</normaloff>:/image/image/userIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAchieve">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>116</width>
     <height>56</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>116</width>
     <height>56</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>116</width>
     <height>56</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string> 成 就</string>
   </property>
   <property name="icon">
    <iconset resource="imandqss.qrc">
     <normaloff>:/image/image/achiIcon.png</normaloff>:/image/image/achiIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/closeIcon.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>30</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDwi">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>35</width>
     <height>40</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/dwiIcon.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>30</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInstal">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>34</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/installcon.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>34</height>
    </size>
   </property>
   <property name="shortcut">
    <string>`</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="MainWindow">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>635</width>
     <height>498</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>635</width>
     <height>498</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>635</width>
     <height>498</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="AllGame">
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>141</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 127);
    border: none;
     max-height: 1px;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>161</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 127);
</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
position: absolute;
width: 55px;
height: 35px;
top: 295px;
left: 76px;


color: rgb(255, 255, 255)


</string>
     </property>
     <property name="text">
      <string>推 荐：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 记 录 */
position: absolute;
width: 55px;
height: 35px;
top: 295px;
left: 76px;


color: rgb(255, 255, 255)
</string>
     </property>
     <property name="text">
      <string>所 有 游 戏：</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>601</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
 background: transparent; </string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>142</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="TZFE">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>100</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/game/assets/2048/2048.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Snake">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>100</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/game/assets/Snake/logo.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Gobang">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>100</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/game/assets/Gobang/logo.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="color">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>100</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/game/assets/GuessColor/logo.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="comingsoom">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/image/image/comingsoon.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>615</width>
       <height>193</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="imandqss.qrc">:/image/image/1.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="History">
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>621</width>
       <height>491</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <widget class="QWidget" name="tcs">
      <attribute name="title">
       <string>五子棋</string>
      </attribute>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>621</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size: 20px;
font-weight: 400
</string>
       </property>
       <property name="text">
        <string>   NUM   |   BESTSCORE    |    STARTTIME    |   ENDTIME</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>621</width>
         <height>391</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="wzq">
      <attribute name="title">
       <string>贪吃蛇</string>
      </attribute>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>621</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size: 20px;
font-weight: 400
</string>
       </property>
       <property name="text">
        <string>   NUM   |   BESTSCORE    |    STARTTIME    |   ENDTIME</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>621</width>
         <height>391</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>2048</string>
      </attribute>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>621</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size: 20px;
font-weight: 400
</string>
       </property>
       <property name="text">
        <string>   NUM   |   BESTSCORE    |    STARTTIME    |   ENDTIME</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>621</width>
         <height>391</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>色彩达人</string>
      </attribute>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>621</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size: 20px;
font-weight: 400
</string>
       </property>
       <property name="text">
        <string>   NUM   |   BESTSCORE    |    STARTTIME    |   ENDTIME</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>621</width>
         <height>391</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="User">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>头像</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>UserName:</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>150</y>
       <width>351</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 127);
    border: none;
     max-height: 1px;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Account:</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>621</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  background: transparent;</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>619</width>
        <height>785</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="Line" name="line_14">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_44">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font-size: 20px;
font-weight: 400
</string>
         </property>
         <property name="text">
          <string>     NAME       |   BESTSCORE    |   NUMBER    |   GAMETIME</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_15">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_45">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_46">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_47">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_48">
         <property name="text">
          <string>ds



\n

\n










\nn\n










</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_49">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_50">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="achieve">
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>621</width>
       <height>491</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <widget class="QWidget" name="tcs_2">
      <attribute name="title">
       <string>五子棋</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="wzq_2">
      <attribute name="title">
       <string>贪吃蛇</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>2048</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>色彩达人</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Instal">
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>换头像</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 127);</string>
     </property>
     <property name="text">
      <string>改名</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="imandqss.qrc"/>
 </resources>
 <connections/>
</ui>
